mod/01_rating/set/type_plus_right/metric_value/value.rb

```diff
@@ -91,7 +90,10 @@ def metric_name_from_params
 
 event :update_related_scores, :finalize,
       on: [:create, :update, :delete],
-      when: proc { |c| !c.metric_card.scored? } do
+      when: proc { |c|
+        c.metric_card.type_id == MetricID &&
```

Comment:
for this to fail, we would have to have a busted Metric Value, right?  So shouldn't this checking be in the Metric Value type set?